src/process-article.ts

```diff
@@ -0,0 +1,48 @@
+import cheerio from 'cheerio'
+import { readFileSync, writeFileSync } from 'fs'
+import { relative } from 'path'
+
+import {
+  appendFooter,
+  appendHtmlPostfix,
+  replaceANamespaceWithWiki,
+  reworkLinks
+} from './article-transforms'
+import { Directories, EnhancedOpts, Options } from './domain'
+
+export const processArticle = (
```

Comment:
Just to be sure I got this right: we still modify every page?
Is this because we had to fix URLs in links and images and it was not possible to do from JS, or is there more to this?

Reply:
Sorry this is no longer used code I will remove it.